application/common/models/Reset.php

```diff
@@ -310,7 +312,9 @@ public function sendPhone()
     public function isUserProvidedCodeCorrect($userProvided)
     {
         if ($this->type == self::TYPE_METHOD && $this->method->type == Method::TYPE_PHONE) {
-            return \Yii::$app->phone->verify($this->code, $userProvided);
+            return Verification::isPhoneCodeValid($this->code, $userProvided);
+        } else {
```

Comment:
Do we need to make separately handle email methods and unknown methods here like we did in `Method->sendVerification()` above?